tests/model_serving/model_server/storage/pvc/utils.py

```diff
@@ -10,31 +11,64 @@ def create_isvc(
     name: str,
     namespace: str,
     deployment_mode: str,
+    storage_key: str,
     storage_uri: str,
     model_format: str,
     runtime: str,
-    min_replicas: int = 1,
+    max_replicas: Optional[int] = 1,
```

Comment:
"""Optional[X] is equivalent to Union[X, None].""" so should be 
```suggestion
    max_replicas: Optional[int] = None,
```
and in the code, add to the dict if max_replicas is not None